cmd/fluxd/main.go

```diff
@@ -389,13 +409,20 @@ func main() {
 					"notes-ref", *gitNotesRef,
 					"set-author", *gitSetAuthor)
 				checkout = working
+				break
+			}
+
+			tryAgain := time.NewTimer(10 * time.Second)
```

Comment:
I believe you're leaking timers here

Reply:
By not receiving on the channel and/or calling `Stop()`?

Reply:
I'm not sure that's true; what is leaking here? I only reassign `tryAgain` in the case that we received a value on its channel, meaning the timer has served its purpose. I don't see anything in the documentation to suggest that it needs to be explicitly `Stop`ped in order to be gc'ed.

Reply:
(of `time.Ticker`s it is true that you have to `Stop()` them to release resources)